universal-application-tool-0.0.1/test/support/ResourceCreator.java

```diff
@@ -52,6 +53,12 @@ public QuestionDefinition insertQuestionDefinition() {
         .getResult();
   }
 
+  public Program insertProgram(String name, LifecycleStage lifecycleStage) {
```

Comment:
(informative, no action required): these methods should be deprecated in favor for ProgramBuilder. 

I'll mark them deprecated in a PR.